src/ui/editor.js

```diff
@@ -1515,6 +1524,12 @@ class SurveyEditor extends Editor {
       clipboardPasteParser  : 'range',
       clipboardPasteAction  : 'range',
 
+      // pagination
+      pagination             : true,
```

Comment:
please make this configurable via the config.js file. 
depending on the screen size one might find useful a configurable number of rows